app/models/conversation.rb

```diff
@@ -0,0 +1,25 @@
+# == Schema Information
```

Comment:
I think this file does not belong to this ticket, wdyt?

Reply:
A conversation is automatically created once a match is found for a target.